src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -22,65 +22,104 @@ freshnessValidatedDate: never
 
 import browserInstallBrowser from 'images/browser_screenshot-crop_install-browser.webp'
 
-To get started with <InlinePopover type="browser" />, you'll need to add our browser agent to the HTML of the views you want to monitor. The browser agent is a snippet of JavaScript code that monitors the performance of your app/site and sends that data to New Relic. You can add that code manually to your site, or use our APM agents to inject that code for you. 
+To get started with <InlinePopover type="browser" />, add our browser agent to the HTML of the views you want to monitor. The browser agent is a snippet of JavaScript code that monitors the performance of your site and sends that data to New Relic. 
 
 ## Before you begin [#before-start] 
 
-Before starting, you may want to check that your browser app meets our [compatibility and requirements](/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring).
+If you haven't done so, you may want to check that your browser app meets our [compatibility and requirements](/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring).
+
+## Install the browser agent [#install]
+
+There are four ways to install the browser agent. Depending on your specific needs, you can:
+
+* Add the snippet directly to your site's code
+* Use one of our APM agents to inject the code for you
+* Query NerdGraph, our GraphQL API, to configure and retrieve the snippet
+* Instrument multiple apps at once with npm and NerdGraph
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Add snippet
+        </TabsBarItem>
+        <TabsBarItem id="2">
+            Install via APM
+        </TabsBarItem>
+        <TabsBarItem id="3">
+            Use NerdGraph
+        </TabsBarItem>
+        <TabsBarItem id="4">
+            Use npm
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="1">
+## Add the snippet directly [#add-snippet]
 
-Want to use our API to programmatically monitor your applications? See [Browser monitoring with NerdGraph](/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph).
-
-## Install the agent [#install]
+<img
+  title="Install browser"
+  alt="Install browser"
+  src={browserInstallBrowser}
+/>
 
-To install the browser agent: Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > Integrations & Agents > Browser and mobile > Browser monitoring**</DNT> and follow the instructions. 
+Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > Add data > Browser and mobile > Browser monitoring**</DNT> and follow the instructions to generate the browser JavaScript snippet in the UI and add it to your site's code. 
```

Comment:
```suggestion
Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > Integrations & Agents > Browser and mobile > Browser monitoring**</DNT> and follow the instructions to generate the browser JavaScript snippet in the UI and add it to your site's code. 
```

Reply:
This is a new change introduced in https://github.com/newrelic/docs-website/pull/18021

Reply:
Oh, of course! Thanks for catching this @ally-sassman!

Reply:
Resolved in my latest commit